### DISCUSSION TITLE
Share this on CiscoDevNet

### DISCUSSION THREAD

**Discussion by tybruno** [scrapli/scrapli_cfg#11] [Ideas]
Hello,

This is an awesome project! I was thinking you could have this shared on [Cisco DevNet](https://developer.cisco.com/codeexchange/explore). I see that [Scrapli](https://developer.cisco.com/codeexchange/github/repo/carlmontanari/scrapli) is also on Cisco DevNet but it seems like the info is a little outdated. I think it would be awesome if all of Scrapli's plugins were on Cisco DevNet.

Let me know what you think. Also, let me know if I can help in some way!

Tyler Bruno

**Comment by carlmontanari** [comment]
Heyo! I almost forgot about this. Yeah probably a good call to add this and the other repos there. Will try to remember to do that this weekend!

I think the core scrapli one got updated too… didn’t realize I had to login to make that happen (I assume cuz it needs my auth via git)
